lib/Math/Constants.pm6

```diff
@@ -9,6 +9,7 @@ my constant c is export = 299792458;
 my constant G is export = 6.67408e-11;
 my constant eulernumber-e is export = 2.7182818284;
 my constant pi is export = 3.142857;
+my constant Euler-Mascheroni-gamma is export = 0,57721566490153286060;
```

Comment:
Please use the English convention for decimal point, not the Spanish one. This results in an error. After you do, please wait a bit for the test result.